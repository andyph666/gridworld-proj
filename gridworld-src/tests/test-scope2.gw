int a = 10;
int b = 15;
print(a);
print(b);

int foo(int sum){
	int a = 5;
	int b = 10;
	a = a + 1;
	b = b + 1;
	sum = a + b;
	print(a);
	print(b);
	print(sum);
}

foo(0);