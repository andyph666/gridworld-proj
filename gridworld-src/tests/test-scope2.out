10
15
6
11
17